soroban-sdk/src/storage.rs

```diff
@@ -274,11 +274,11 @@ impl Persistent {
         self.storage.set(key, val, StorageType::Persistent)
     }
 
-    pub fn bump<K>(&self, key: &K, low_expiration_watermark: u32, high_expiration_watermark: u32)
+    pub fn extend<K>(&self, key: &K, low_expiration_watermark: u32, high_expiration_watermark: u32)
```

Comment:
These arguments should be renamed to `threshold` and `extend_to`.

Reply:
Done